airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_message.yaml

```diff
@@ -118,10 +118,43 @@ definitions:
       name:
         type: string
         description: Stream's name.
-      json_schema:
-        description: Stream schema using Json Schema specs.
+      discovered:
+        description: Contains information about the data in the source. This field is set by the source and should be read-only thereafter.
         type: object
-        existingJavaType: com.fasterxml.jackson.databind.JsonNode
+        properties:
+          json_schema:
+            description: Stream schema using Json Schema specs.
+            type: object
+            existingJavaType: com.fasterxml.jackson.databind.JsonNode
+          metadata:
+            description: Additional information about the data in the source, including information about what Airbyte features it supports.
+            type: object
+            properties:
+              supported_sync_modes:
+                type: array
+                items:
+                  "$ref": "#/definitions/SyncMode"
+                # backwards compatibility: if not present, assume full_refresh only.
+                default:
+                  [full_refresh]
+              default_comparable:
+                description: Path to the field that will be used to determine if a record is new or modified since the last sync. If not provided by the source, the end user will have to specify the comparable themselves.
+                type: array
+                items: string
+      configuration:
```

Comment:
Maybe `sync_configuration`? "A stream's sync configuration" makes more sense to me than just "a stream's configuration", since the catalog is technically part of the configuration.

Reply:
how about `extraction_configuration`? very ELT.

Reply:
actually sync_configuration is fine.

Reply:
fine with either

Reply:
moot with structure change.